src/model/constraints/behavior_constraints.py

```diff
@@ -0,0 +1,31 @@
+from src.model.constraints.constraint import Behavior_Con
+
+class Danger_Con(Behavior_Con):
+    pass
+
+class Does_Maneuver_Con(Behavior_Con):
+
+    def __init__(self, parent, actors, maneuver):
+        self.arity = 2
+        # self.type_id = ? # TODO
+        self.maneuver = maneuver # TODO: Confirm
```

Comment:
I guess we also have to validate maneuver type/IDs at some point?

Reply:
Yes! I think I have some related code in the complete branch.